content/attacks/posts/2022-01-17-Crypto.com.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2022-01-17
+custodians: Crypto.com
+categories: Custodian
+title: Crypto.com Confirms $34M Hack
+---
+
+## Summary
+
+According to the announcement, an attacker found a way to approve transactions without the two-factor authentication control being inputted by account holders. This resulted in 483 Crypto.com users losing funds from their accounts. The exchange reaffirmed comments made by the firm’s CEO, Kris Marszalek, that any accounts found to be impacted were fully restored, resulting in no loss of funds for users.
+
+## Attackers
+
+The identity of the attackers remained unknown.
```

Comment:
add information on theories, sophistication, anything that can narrow the pool of suspects

Reply:
can you mention Lazarus and link to the source, ErgoBTC seems solid researcher